src/pages/DetailsPage.js

```diff
@@ -98,11 +100,25 @@ const DetailsPage = (props) => {
                 {details ? (
                     <ScrollView>
                         <View style={styles.pageWrapper}>
-                            <Avatar
-                                containerStyles={[styles.avatarLarge, styles.mb3]}
-                                imageStyles={[styles.avatarLarge]}
-                                source={details.avatar}
-                            />
+                            <AttachmentModal
+                                sourceURL={details.avatarHighResolution}
```

Comment:
Can't we do that directly here?

Reply:
`details.avatar` being used here is avatar returned from `formatPersonalDetails` function which is of lower resolution. So, new property for higher resolution is added in `formatPersonalDetails` function which is later used here.